Config.uk

```diff
@@ -15,6 +15,9 @@ config UK_ARCH
 	default "arm64"  if ARCH_ARM_64
 	default "arm"	if ARCH_ARM_32
 	default "$(UK_ARCH)"
+config HOST_ARCH
```

Comment:
Nit: Missing newline before the new config directive.

Reply:
Right, I'll add.

Reply:
Okay, I did not put it because it was part of a block without spaces. I am updating with this commit this particular block to add spaces in betweeen.